src/solver.jl

```diff
@@ -32,12 +32,12 @@ end
 
 
 """
-    waveSpeed(A :: Float64, gamma :: Float64)
+    waveSpeed(A :: Float64, gamma :: Float64, Ac :: Float64)
 
-Compute pulse wave velocity at the given node.
+Compute pulse wave velocity at the given node. It includes the catheter.
 """
-function waveSpeed(A :: Float64, gamma :: Float64)
-    return sqrt(3*gamma*sqrt(A)*0.5)
+function waveSpeed(A :: Float64, gamma :: Float64, Ac :: Float64)
```

Comment:
cambiando la definizione di questa funzione alcuni test non passano

per semplificare, puoi dare a `Ac` un valore di default (direi `0.0`) che viene usato nel caso in cui la funzione sia chiamata senza quell'argomento

```
function waveSpeed(A::Float64, gamma::Float64, Ac::Float64 = 0.0)
```

Reply:
oppure cambi i test aggiungendo `Ac` nella chiamata